libraries/bsg_manycore_eva.cpp

```diff
@@ -50,18 +50,18 @@
 #define DEFAULT_GROUP_BITIDX (DEFAULT_GROUP_Y_BITIDX + DEFAULT_GROUP_Y_LOGSZ)
 #define DEFAULT_GROUP_BITMASK (1ULL << DEFAULT_GROUP_BITIDX)
 
-#define DEFAULT_GLOBAL_X_LOGSZ 6
-#define DEFAULT_GLOBAL_X_BITIDX HB_MC_EPA_LOGSZ
+#define DEFAULT_GLOBAL_X_LOGSZ 7
```

Comment:
Can't this be replaced by rom value?

Reply:
All of this garbage could probably be replaced with ROM values. Is that a good use of our time right now?

Reply:
This one in particular is in the rom, no? The x/y coordinate width?

Reply:
Kinda. There's also a group width which is inexplicably only 6. The relation between this value and the x-coord width does not feel like 'solid ground' to me.

Reply:
That suggests there should be a group width field in the rom... is the group with just implicit?